### PR TITLE
Updated Design

### PR DESCRIPTION
**Pull request by danglakshay**
#97  Updated Design for Nav bar, User Dashboard, and Admin Nav bar.
#120   Code also features a flask integrated registration flow.

### CODE REVIEW

File: code/login.py
Comment (noahstaz) [inline code/login.py:1]: This should be done in the app.py folder and then directed to from the main page to ensure that all our code is ran out of the same base folder and then redirected to, check out how I did this for alerts on my pr #117

**Review by noahstaz — CHANGES_REQUESTED** [review]
I think everything code wise is good, but you should make that change before merging into main for easier code consistency

**Review by komalsingh0925 — APPROVED** [review]
Nice work!

**Review by noahstaz — APPROVED** [review]
Changes look good!

**Pull request merged**